examples/training/semantic_segmentation/pascal_voc/deeplab_v3.py

```diff
@@ -129,12 +136,14 @@ def proc_eval_fn(examples):
         boundaries=[30000 * 16 / global_batch],
         values=[base_lr, 0.1 * base_lr],
     )
-    model = DeepLabV3(
-        classes=21,
-        backbone="resnet50_v2",
+    backbone = ResNet50V2(
         include_rescaling=True,
-        backbone_weights="imagenet",
+        stackwise_dilations=[1, 1, 1, 2],
```

Comment:
Please also add a comment here, 
"the stackwise_dilation config improves the meanIOU by 2 points"

Reply:
Given I have verified the difference is indeed cause by the config. We should document it here so future users wouldn't make the same mistake.
Also we should start to think how to pass backbone configs into a generic basic_training script.
@DavidLandup0

Reply:
Added!
Yeah, I was thinking about the same thing. Doing the backbone args might prove to be a pain if we want to do them in a user friendly way.
I suppose we could do a list arg, and unpack it but that feels a bit ragged.

I'll post some proposals in the general script once this and DLV3Plus are done